content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -113,7 +113,7 @@ actions. Failures defined by the `failurePolicy` are enforced
 according to these actions only if the `failurePolicy` is set to `Fail` (or not specified),
 otherwise the failures are ignored.
 
-See [Audit Annotations: validation falures](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation_failure)
+See [Audit Annotations: validation failures](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation_failure)
```

Comment:
Looking at the [preview page](https://deploy-preview-43893--kubernetes-io-main-staging.netlify.app/docs/reference/access-authn-authz/validating-admission-policy/), it seems that the link itself is a broken link that needs fixing, along with a minor typo in the link text. Please see the suggestions below. 
_Also, it would be great if you could also update the PR description for other reviewer/approvers._

```suggestion
See [Audit Annotations: validation failures](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation-failure)
```

Reply:
oops didn't realize this. Nice catch. Will apply in the suggestions. 
Thanks for the review!